### PR TITLE
:barber: more frontend changes

### PR DESCRIPTION
**Pull request by ajthomascouk**
Creating a PR so changes can be demoed.

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

**Review by markysoft — CHANGES_REQUESTED** [review]
Seems a bit strange having a class called 'h3', and applying it to an h2 element.

Should it be called something more descriptive, like 'sub-title'?

**Comment by ajthomascouk** [comment]
@markysoft I'm not to fussed what its called. I think it's clearer what we're trying to achieve from the style if its call '.h3' If we call it 'sub-title' for example, is it a H3 style or is it something else?

File: scss/c2s/c2s.scss [hunk 1]
```diff
@@ -1,6 +1,13 @@
 @import "betahealth/nhsuk";
 
 /*
+   Colours.
+*/
+$hiviz: #ffca55;
+$blue: #0072c6;
```
Comment (st3v3nhunt) [inline scss/c2s/c2s.scss]: If the variable is defined already within the `@import` does that make it available within this? It looks like it has been defined already in https://github.com/nhsuk/profiles/blob/db1e8a00ca69b5feb3b692a360d081670b2d9456/scss/betahealth/environment/settings/_colours.scss#L1

File: scss/c2s/c2s.scss [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+$callout-border-blue: #40ADD7;
```
Comment (st3v3nhunt) [inline scss/c2s/c2s.scss]: I can't see the usage of `$callout-border-blue` or `$callout-blue`. Should the be used at https://github.com/nhsuk/profiles/pull/30/files#diff-fbb593c07f7b6259a85d6bb2324072c7R108 ?

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]

**Comment by st3v3nhunt** [comment]
@ajthomascouk I agree with @markysoft about the naming of the `h3` class on the `<h2>`. Seems like it might make it harder in the future to understand what classes should be used where.

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

File: scss/c2s/c2s.scss (hunk 1 above)
Comment (ajthomascouk) [reply scss/c2s/c2s.scss]: You're not wrong!

File: scss/c2s/c2s.scss (hunk 2 above)
Comment (ajthomascouk) [reply scss/c2s/c2s.scss]: `106 .callout--blue {`
`107   background-color: rgba(64, 173, 215, 0.1);`
`108   border-left: 8px solid #40ADD7;`
`109   padding: 8px 16px;`
`110 }`

Should have been used here, but i put the values in for some reason.

File: scss/c2s/c2s.scss (hunk 1 above)
Comment (ajthomascouk) [reply scss/c2s/c2s.scss]: > It looks like it has been defined already in 

Removed from our SCSS

**Comment by ajthomascouk** [comment]
@st3v3nhunt wrote:
> I agree with @markysoft about the naming of the h3 class on the `<h2>`

Okay, so sticking with BEM, how about something like `.h2--smaller`?

**Comment by st3v3nhunt** [comment]
Re: https://github.com/nhsuk/profiles/pull/30#issuecomment-282705481

@ajthomascouk Sounds good to me. Should be good for another review once that has been done?
/cc @markysoft

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-30.dev.c2s.nhschoices.net](http://profiles-pr-30.dev.c2s.nhschoices.net)

**Review by st3v3nhunt — APPROVED** [review]

**Comment by st3v3nhunt** [comment]
@markysoft Looks like you need to approve the changes if you have previous requested changes. I'm happy with the changes of [4ea3154](https://github.com/nhsuk/profiles/pull/30/commits/4ea315496dfe828273ab31c428c4f67e007546fb)

**Review by markysoft — APPROVED** [review]

**Pull request merged**